components/test-run.html

```diff
@@ -0,0 +1,77 @@
+<!--
+Copyright 2017 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<link rel="import" href="/bower_components/polymer/polymer-element.html">
+<link rel="import" href="/bower_components/polymer/lib/elements/dom-if.html">
+<link rel="import" href="/bower_components/polymer/lib/elements/dom-repeat.html">
+
+<!--
+`<test-run>` is a stateless component for displaying the details of a TestRun.
+
+The schema for the testRun property is as follows:
+{
+  "browser_name": "",
+  "browser_version": "",
+  "os_name": "",
+  "os_version": "",
+  "revision": "",     // the first 10 characters of the SHA
+  "created_at": "",   // the date the TestRun was uploaded
+}
+
+See models.go for more details.
+-->
+<dom-module id="test-run">
+  <template>
+    <style>
+      :host {
+        display: block;
+        font-size: 16px;
+      }
+      img {
+        width: 32px;
+        height: 32px;
+      }
+    </style>
+
+    <div>
+      <div><img src="/static/{{testRun.browser_name}}_64x64.png" /></div>
+      <div>{{testRun.browser_name}} {{testRun.browser_version}}</div>
+      <div>{{testRun.os_name}} {{testRun.os_version}}</div>
+      <div>@{{testRun.revision}}</div>
+      <div>{{dateFormat(testRun.created_at)}}</div>
+    </div>
+  </template>
+
+  <script>
+    class TestRun extends window.Polymer.Element {
+      static get is () { return 'test-run' }
+
+      static get properties () {
+        return {
+          testRun: {
```

Comment:
One thought: the fact that testRun is a property whereas TestRun is the class name confused me at first; one possible alternative might be to have 4 properties (browserName, browserVersion, revision, createdAt). Although this might make the declaration of the <test-run> element in test-file-results.html more complicated.